index.html

```diff
@@ -21,7 +21,7 @@ <h1 class="heading-1" >Decorus</h1>
             <input type="text" placeholder="Search..">
 
             <ul class="nav-menu">
-                <li><button class="btn "><i class="fas fa-user fa-lg"></i></button></li>
+                <li><button class="btn "><a class="link" href="/authentication/login.html"><i class="fas fa-user fa-lg"></i></a></button></li>
 
```

Comment:
Not sure if there was a specific reason but  anchor inside a button is not a very good practice, you could use either anchor tag or a button tag , you can just put the class="btn" inside the anchor tag if the style is what u need